modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/file/AbstractFileIO.java

```diff
@@ -57,14 +57,29 @@ private int fully(IOOperation operation, long position, int num, boolean write)
                     time = 0;
                 }
                 else if (n == 0 || i > 0) {
+                /* Condition for reaching the end of file while there's still space in buffer
```

Comment:
Which buffer do you mean - this method doesn't operate buffer. Actually, this code doesn't need in comments, it is pretty straightforward. You can make this code better if add javadoc for the return value

Reply:
Commentary removed, javadoc for the return value added.